Swiftfin tvOS/Views/ItemView/Components/OverviewView.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2025 Jellyfin & Jellyfin Contributors
+//
+
+import JellyfinAPI
+import SwiftUI
+
+// TODO: have items provide labeled attributes
+// TODO: don't layout `VStack` if no data
+
+extension ItemView {
+
+    struct OverviewView: View {
+
+        let item: BaseItemDto
+        private var overviewLineLimit: Int?
+        private var taglineLineLimit: Int?
+
+        var body: some View {
+            VStack(alignment: .leading, spacing: 10) {
+
+                if let firstTagline = item.taglines?.first {
+                    Text(firstTagline)
+                        .font(.subheadline)
+                        .fontWeight(.semibold)
+                        .multilineTextAlignment(.leading)
+                        .lineLimit(taglineLineLimit)
+                }
+
+                if let itemOverview = item.overview {
+                    Text(itemOverview)
+                        .font(.subheadline)
+                        .lineLimit(overviewLineLimit)
+                }
+
+                if let birthday = item.birthday?.formatted(date: .long, time: .omitted) {
```

Comment:
This looks unbalanced with the overview above these values, let's move them above the overview/tagline.